examples/stocks/lib/main.dart

```diff
@@ -94,6 +94,13 @@ class StocksAppState extends State<StocksApp> {
     return null;
   }
 
+  Future<LocaleQueryData> _onLocaleChanged(ui.Locale locale) {
+    return initializeMessages(locale.toString()).then((_) {
+      Intl.defaultLocale = locale.toString();
+      return StockStrings.instance;
```

Comment:
It's odd that we get this value from a static instead of having the future resolve to the instance.